packages/discussions/graphql/resolvers/_queries/activeDiscussions.js

```diff
@@ -0,0 +1,24 @@
+const moment = require('moment')
+
+module.exports = async (_, { lastDays = 5 }, { pgdb }) =>
+  pgdb.query(`
+    SELECT
+      COUNT(*) as count,
+      d.*
+    FROM comments c
+    JOIN
+      discussions d
+      ON c."discussionId" = d.id
```

Comment:
Can we update this join to *not* include *closed* discussions? FYI, we just decided to close some catch-all discussion at launch-time, so it would be good to ignore them here already.

Reply:
sure we can ;)